### PR TITLE
Adding Sardinian

### PR DESCRIPTION
**Pull request by adrmzz**
Updated jitsi-meet/react/features/base/i18n/BuiltinLanguages.native.js to add Sardinian

### CODE REVIEW

**Comment by adrmzz** [comment]
Hi! I already did it for the last contribution. Should I do it again?

El 14/4/20 a les 12:00, jitsi-jenkins ha escrit:

**Comment by damencho** [comment]
Could you please fix the failing lint checks.

**Comment by adrmzz** [comment]
Should be OK now.

File: lang/main-sc.json [hunk 1]
```diff
@@ -220,7 +220,7 @@
         "muteParticipantDialog": "Seguru chi boles pònnere custa persone a sa muda? No as a pòdere torrare a ativare s'àudio issoro, però is utentes ddu ant a pòdere ativare in cale si siat momentu.",
         "muteParticipantTitle": "Boles pònnere custa persone a sa muda?",
         "Ok": "AB",
-        "passwordLabel": "Unu partetzipante at blocadu sa riunione. Inserta sa $t(lockRoomPassword) pro intrare.",
+        "passwordLabel": "$t(lockRoomPasswordUppercase)",
```
Comment (damencho) [inline lang/main-sc.json:223]: Actually, this in English is:
`"passwordLabel": "The meeting has been locked by a participant. Please enter the $t(lockRoomPassword) to join.",`
Can you change it?

**Comment by damencho** [comment]
Thank you

**Comment by adrmzz** [comment]
Thank you!

**Pull request merged**